lib/plugins/Label_HX.ts

```diff
@@ -0,0 +1,66 @@
+import { DecoderPlugin } from '../DecoderPlugin';
+import { DecodeResult, Message, Options } from '../DecoderPluginInterface';
+import { CoordinateUtils } from '../utils/coordinate_utils';
+import { ResultFormatter } from '../utils/result_formatter';
+
+export class Label_HX extends DecoderPlugin {
+  name = 'label-hx';
+
+  qualifiers() { // eslint-disable-line class-methods-use-this
+    return {
+      labels: ['HX'],
+      preambles: ['RA FMT LOCATION', 'RA FMT 43'],
+    };
+  }
+
+  decode(message: Message, options: Options = {}) : DecodeResult {
+    const decodeResult = this.defaultResult();
+    decodeResult.decoder.name = this.name;
+    decodeResult.message = message;
+    decodeResult.formatted.description = 'Undelivered Uplink Report';
```

Comment:
missed this last time - is this the right name?

Reply:
not really sure but I grabbed it from here: https://raw.githubusercontent.com/airframesio/data/master/json/acars/metadata.json

Reply:
works for me!